app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,262 @@
+
+```shell
+kubectl exec -n first-consumer consumer -- curl --no-progress-meter demo-app.kuma-demo.mesh:5000/counter
+```
+
+You should see something similar to:
+
+```json
+{"counter":"0","zone":"local","err":null}
+```
+
+## Namespace scoped policies
+
+Now that we have our setup we can start playing with policies. Let's create simple MeshTimeout policy in `kuma-demo` namespace:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: producer-timeout
+  namespace: kuma-demo
+  labels:
+    kuma.io/mesh: default
+    kuma.io/origin: zone
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: demo-app
+      default:
+        http:
+          requestTimeout: 1s" | kubectl apply -f -
+```
+
+We can now inspect applied policy labels:
+
+```shell
+kubectl get meshtimeout -n kuma-demo producer-timeout -o jsonpath='{.metadata.labels}'
+```
+
+We should see output like:
+
+```json
+{
+  "k8s.kuma.io/namespace": "kuma-demo",
+  "kuma.io/env": "kubernetes",
+  "kuma.io/mesh": "default",
+  "kuma.io/origin": "zone",
+  "kuma.io/policy-role": "producer",
+  "kuma.io/zone": "default"
+}
+```
+
+{{site.mesh_product_name}} is adding custom labels to our policy. The one that interests us most at the moment is `kuma.io/policy-role`.
+This is new label that indicates policy role. Possible values of this label are:
+
+- `system` - policies applied in system namespace
+- `workload-owner` - policies that specify `spec.from` section
+- `consumer` - policies targeting MeshServices from different namespace
+- `producer` - policies targeting MeshServices from the same namespace
+
+### Producer consumer model
+
+With namespace scoped policies we've introduced producer/consumer model for policies. 
+
+**Producer** is someone who authors and operates service. Producer can create default policies that will be used when communicating with service. 
+Producer policies will be created in the same zone as MeshService they target. Producer policies will be synced to other zones. 
+
+**Consumer** is someone that utilizes a service by communicating with it. Consumer policies will be applied in consumer 
+namespace and will target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
+Consumer policies will override producer policies.
+
+## Testing producer policy
+
+To test MeshTimeout that we've applied in previous steps we need to simulate delays on our requests. To do this we need
+MeshFaultInjection policy
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshFaultInjection
+metadata:
+  name: default
+  namespace: kuma-demo
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: [Sidecar]
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          - delay:
+              percentage: '50'
+              value: 2s" | kubectl apply -f -
+```
```

Comment:
I'm on the fence with using the Mesh to test the Mesh that's also something @johnharris85 found it confused users.

Maybe we should just add an option to delay the response in the demo app?

Reply:
Yeah, I not a big fan of testing this with fault injection but this is what we did previously. I've added possibility to specify delay and specific status code in kuma-counter-demo: https://github.com/kumahq/kuma-counter-demo/pull/53